tcad_bot/handlers.py

```diff
@@ -30,8 +37,12 @@ def get_uv_index(bot, update):
         filter(lambda p: p['nombre'] == 'SANTIAGO', radiation_data)
     )
 
+    date = radiation_stgo['fechapron']
+    index = radiation_stgo['indicepron'].split(':')
+
+    text = 'Pronostico Dia: {}. UV index: {}({})'.format(date, index[0], index[1])
```

Comment:
This can be done like this:

    'Pronostico Dia: {0}. UV index: {1[0]}({1[1]})'.format(date, index)

This way yo pass less parameters ;)